events-processor/processors/processors.go

```diff
@@ -85,7 +87,7 @@ func initFlagStore(name string) (*models.FlagStore, error) {
 		Address:  os.Getenv(envLagoRedisStoreURL),
 		Password: os.Getenv(envLagoRedisStorePassword),
 		DB:       redisDb,
-		UseTLS:   os.Getenv(envEnv) == "production",
+		UseTLS:   utils.GetEnvAsBool(envLagoRedisStoreTLS, false),
```

Comment:
It changes the behavior when env variable is not set and  env is `production`.

Maybe we can fallback to the existing behavior when the env var is not set:
```suggestion
		UseTLS:   utils.GetEnvAsBool(envLagoRedisStoreTLS, os.Getenv(envEnv) == "production"),
```

Reply:
I already added this new env var on our Cloud, this should not bring any issue?